kv/range_cache.go

```diff
@@ -228,7 +236,14 @@ func (rdc *rangeDescriptorCache) getCachedRangeDescriptorLocked(key proto.Key) (
 
 	// Check that key actually belongs to range
 	if !rd.ContainsKey(keys.KeyAddress(key)) {
-		return nil, nil
+		if !isReverse {
```

Comment:
i don't understand this change; does the reverse scan change the edge key inclusion rule?